pyk/src/pyk/ktool/kprove.py

```diff
@@ -115,17 +111,15 @@ def prove(self, specFile, specModuleName, args=[], haskellArgs=[], logAxiomsFile
         command += args
         commandEnv = os.environ.copy()
         commandEnv['KORE_EXEC_OPTS'] = ' '.join(haskellArgs + haskellLogArgs)
-        notif(' '.join(command))
+        _LOGGER.info(' '.join(command))
         process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True, env=commandEnv)
         stdout, stderr = process.communicate(input=None)
         try:
             finalState = KAst.from_dict(json.loads(stdout)['term'])
         except Exception:
-            sys.stderr.write(stdout + '\n')
-            sys.stderr.write(stderr + '\n')
-            fatal(f'Exiting: process returned {process.returncode}')
+            raise RuntimeError(f'Process returned {process.returncode}')
```

Comment:
We should include the stdout/stderr in the output somehow, via logging perhaps, at `logging.error(...)` level.

Getting that output is necessary for understanding how to fix the proof claim we're feeding to the backend.

Reply:
We should probably pass this information along with the exception. Then the caller can decide what to do with that data.

Eventually, we should introduce a class `KProveError` (or a hierarcy of such classes) that stores the information extracted from the output of `kprove` in a structured form.

Reply:
That sounds good, let's do `f'Process returned {process.returncode}\nStdout:\n{stdout}\nStderr:\n{stderr}'` for now I think. I made a note on the project board about introducing an error class heirarchy.

Reply:
Done.